src/debezium-service/templates/pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
```

Comment:
Wrap it in {{ if .Values.global.ha}}
-- content 

{{ - end }}

take a look into other pdbs